[BrowserKit] Add response management to BrowserKit documentation (without toArray) (#16949)
symfony/symfony-docs: components/browser_kit.rst

        '.table-list-header-toggle a:nth-child(1)'
    )->text());

Dealing with HTTP responses
~~~~~~~~~~~~~~~~~~~~~~~~~~~

When using the BrowserKit component, you may need to deal with responses of

> the requests you made. To do so, you simply have to make a request, then call

Reviewer: ```suggestion
the requests you made. To do so, you need to call
```

Author: Thank you for the suggestion!
About version 4.4, the `getResponse()` method didn't exist at this time. That's why I targeted 5.4 :slightly_smiling_face:

Reviewer: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/BrowserKit/Client.php#L234 🤔

Author: Oooh that was moved to `AbstractBrowser` after, I did not check `Client`! Thank you, I'm rebasing!
